repositories.json

```diff
@@ -1,4 +1,5 @@
 {
+  "WildCardFNV": "https://raw.githubusercontent.com/PsychoticNM/Wild-Card-FNV/main/modlists.json",
```

Comment:
For legacy sake, we ask you to put your repository below / last, and not at the top.